### DISCUSSION TITLE
Improve the search functionality of the website

### DISCUSSION THREAD

**Discussion by vmarquar** [community-scripts/ProxmoxVE#434] [General]
While searching for a specific script, I stumbled accross an inconsistency in the search function of the new website.

For example, a search for `homeassistant` returns all items with `home assistant`:
<img width="951" alt="Bildschirmfoto 2024-11-22 um 16 27 06" src="https://github.com/user-attachments/assets/bed1c0dd-e2f5-4079-8e93-9bc6ca3b4d0b">

Searching for `haos` gives `Home Assistant OS`
<img width="951" alt="Bildschirmfoto 2024-11-22 um 16 26 48" src="https://github.com/user-attachments/assets/ba7f39e6-c01b-4535-a1fe-382e52a19c52">

You can also search for `Prox` which results in only 1 entry `NGINX Proxy Manager` --> all the other Entries containing `Proxmox` are omitted.

When I inspected the network tab, it seems like the search is only filtering for the `slug` entry of the json file that comes from the `categories` API-endpoint ([--> endpoint](https://community-scripts.github.io/ProxmoxVE/api/categories)).

My suggestion would be to improve the search functionality by filtering for the `slug` and `name` attribute of the json.

Might be related to [this discussion as well](https://github.com/community-scripts/ProxmoxVE/discussions/220).

**Comment by vmarquar** [comment]
I think this is the line that needs an edit: 
[                  value={script.slug}](https://github.com/community-scripts/ProxmoxVE/blob/5eaee7d6378cc70ddda98651313dfa3ebb7bceb6/frontend/src/components/CommandMenu.tsx#L98)

**Comment by havardthom** [comment]
Thanks for reporting this, PR for fix https://github.com/community-scripts/ProxmoxVE/pull/731
